composer.json

```diff
@@ -14,8 +14,8 @@
         }
     ],
     "require": {
-        "php": "^5.3.9 || ^7.0",
-        "nikic/php-parser": "^1.3",
+        "php": ">=5.5.9 || ^7.0",
```

Comment:
Shouldn't this be `^5.5.9 || ^7.0`? When using `>=5.5.9`, PHP 7.x is allowed anyway (and the same will be true for PHP >= 8.0).